lib/python/vdm/server/HTTPListener.py

```diff
@@ -1617,10 +1613,6 @@ def main(runner, amodule, config_dir, server):
                      view_func=START_LOCAL_SERVER_VIEW, methods=['PUT'])
     APP.add_url_rule('/api/1.0/databases/<int:database_id>/servers/recover',
                      view_func=RECOVER_DATABASE_SERVER_VIEW, methods=['PUT'])
-
```

Comment:
This is still leaving other api/1.0/deployment endpoints. We should use api/1.0/databases/{id}/deployment endpoints instead.